packages/transaction-controller/src/types.ts

```diff
@@ -42,6 +42,7 @@ type TransactionMetaBase = {
   blockNumber?: string;
   deviceConfirmedOn?: WalletDevice;
   verifiedOnBlockchain?: boolean;
+  estimatedBaseFee?: string;
```

Comment:
Just a reminder to keep this in order when merging with `develop`.

Reply:
Sure thing, that's why added to bottom